src/storage.rs

```diff
@@ -25,7 +25,7 @@ where
         // Furthermore, we also have to wait for the countdown before reading the eeprom again.
         // Basically, we have to wait before any I2C access and ensure that the countdown is
         // running again afterwards.
-        count_down.start(Duration::from_millis(0));
+        count_down.start(Duration::from_millis(5));
```

Comment:
Any reason for this change?

Reply:
I guess @eldruin is right and the initial countdown should be zero. The rationale is already given in the comment above.